ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.alp320

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.20
+
+RUN apk --update add bash shadow openssh-server openssh-client unzip zip wget git curl make gcc perl xvfb \
+        libxrender libxi libxtst procps musl-dev perl-doc alsa-lib libx11 msttcorefonts-installer fontconfig libxext freetype zlib fakeroot gnupg
+
+# Add SSL Test packages
+RUN apk --update add openssl gnutls gnutls-dev gnutls-utils nss nss-dev nss-tools
+
+RUN ssh-keygen -t rsa -f /etc/ssh/ssh_host_rsa_key -P ""
+
+## Ensure Fonts Are Updated (Issue https://github.com/adoptium/infrastructure/issues/3039)
+RUN update-ms-fonts
+
+# Get latest jdk21 ga
+RUN wget -q 'https://api.adoptium.net/v3/binary/latest/21/ga/alpine-linux/x64/jdk/hotspot/normal/eclipse?project=jdk' -O /tmp/jdk21.tar.gz
+RUN gpg --keyserver keyserver.ubuntu.com --recv-keys 3B04D753C9050D9A5D343F39843C48A565F8F04B
+# Get sig file for latest jdk21 ga
+RUN wget -q `curl -s 'https://api.adoptium.net/v3/assets/feature_releases/21/ga?architecture=x64&heap_size=normal&image_type=jdk&jvm_impl=hotspot&os=alpine-linux&page=0&page_size=1&project=jdk&vendor=eclipse' | grep signature_link | awk '{split($0,a,"\""); print a[4]}'` -O /tmp/jdk21.sig
+RUN gpg --verify /tmp/jdk21.sig /tmp/jdk21.tar.gz
+RUN mkdir -p /usr/lib/jvm/jdk21 && tar -xpzf /tmp/jdk21.tar.gz -C /usr/lib/jvm/jdk21 --strip-components=1
+
+# Install ant and ant-contrib.
+RUN wget -O /tmp/ant.zip 'https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.9-bin.zip'
```

Comment:
Prefer 1.10.14

Reply:
Not that I disagree with this but I feel if we are going to upgrade the ant version we should have that as a separate issue and run it through appropriate testing